src/styles/blocks/main.scss

```
@@ -0,0 +1,20 @@
+.products {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  max-width: 944px;
+  padding-top: 50px;
+  padding-bottom: 48px;
```

```suggestion
  padding-top: 50px;
  padding-bottom: 50px;
```
cards container should have fixed paddings (see 1024px example), bottom the same as top
this is the task condition